packages/grafana-ui/src/components/VizTooltip/SeriesTable.story.tsx

```diff
@@ -4,7 +4,7 @@ import React from 'react';
 import { SeriesTable } from './SeriesTable';
 
 const meta: Meta<typeof SeriesTable> = {
-  title: 'VizTooltip/SeriesTable',
+  title: 'Visualizations/SeriesTable',
```

Comment:
```suggestion
  title: 'Visualizations/VizTooltip',
```
@JoaoSilvaGrafana I think this is the correct way of doing this, correct? (not as familiar with storybook)

Reply:
We should probably keep the name of the component from the story so actually maybe `Visualizations/VizTooltip/SeriesTable` is the best, even though it is a folder inside a folder